addons/sys_core/stringtable.xml

```diff
@@ -1,6 +1,15 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project name="ACRE">
     <Package name="sys_core">
+        <Key ID="STR_ACRE_sys_core_CategoryUI">
+            <English>ACRE2 UI</English>
+            <German>ACRE2 Nutzeroberfläche</German>
+            <Polish>ACRE2 UI</Polish>
+            <French>ACRE2 IU</French>
+            <Turkish>ACRE2 UI</Turkish>
+            <Japanese>ACRE2 UI</Japanese>
+            <Italian>ACRE2 IU</Italian>
```

Comment:
```suggestion
            <Italian>ACRE2 IU</Italian>
            <Spanish>ACRE2 Interfaz</Spanish>
```

Reply:
Added back to CBA: https://github.com/CBATeam/CBA_A3/pull/1216